spec/spec_helper.rb

```diff
@@ -27,24 +37,24 @@
     %r{\Aspec/tools/} => 'tools/',
   )
 end
-use_headful_chrome = ENV.fetch('HEADFUL_CHROME', nil).present?
+
 require 'factory_bot_rails'
 require 'webmock'
 require 'webmock/rspec'
 require 'pundit/rspec'
-require Rails.root.join('spec/support/fixture_builder.rb').to_s
-Dir['spec/support/**/*.rb'].each { |file| require Rails.root.join(file) }
 # Prevent database truncation if the environment is production
 abort('The Rails environment is running in production mode!') if Rails.env.production?
 require 'rspec/rails'
 require 'capybara/cuprite'
 require 'capybara/rails'
 require 'capybara/rspec'
 require 'capybara/email/rspec'
-require 'capybara-screenshot/rspec' unless use_headful_chrome
+require 'capybara-screenshot/rspec' unless SpecHelper.use_headful_chrome?
 require 'mail'
 require 'percy/capybara'
 require 'super_diff/rspec-rails'
+require Rails.root.join('spec/support/fixture_builder.rb').to_s
+Dir['spec/support/**/*.rb'].each { |file| require Rails.root.join(file) }
```

Comment:
I wanted to move these requires down just because it seems better to have all of the gems loaded before we load our support files, in case our support files want to reference something from one of the gems.